colors/1989.vim

```diff
@@ -166,3 +166,11 @@ call <SID>set_hi("markdownH5", s:light_blue, s:none, "NONE")
 call <SID>set_hi("markdownH6", s:light_blue, s:none, "NONE")
 call <SID>set_hi("markdownHeadingDelimiter", s:light_blue, s:none, "NONE")
 call <SID>set_hi("markdownRule", s:light_blue, s:none, "NONE")
+
+" https://github.com/kien/rainbow_parentheses.vim
+let g:rbpt_colorpairs = [
```

Comment:
Ah, I'm using https://github.com/luochen1990/rainbow, which doesn't use the same variable, so you can just ignore me 🐱